tests/flow/enums-unknown-members/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -0,0 +1,47 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`enum-unknown-members.js - {"trailingComma":"all"} format 1`] = `
+====================================options=====================================
+parsers: ["flow"]
+printWidth: 80
+trailingComma: "all"
+                                                                                | printWidth
+=====================================input======================================
+// @flow
+
+enum E {
+  A,
+  B,
+  ...
+}
+
+=====================================output=====================================
+// @flow
+
+enum E {
+  A,
+  B,
+  ...
+}
+
+================================================================================
+`;
+
+exports[`enum-unknown-members-empty.js - {"trailingComma":"all"} format 1`] = `
+====================================options=====================================
+parsers: ["flow"]
+printWidth: 80
+trailingComma: "all"
+                                                                                | printWidth
+=====================================input======================================
+// @flow
+
+enum E {...}
+
+=====================================output=====================================
+// @flow
+
+enum E {...}
```

Comment:
The `bracketSpacing` option (`true` by default) should be respected here. Cf. `type E = { ... };`

Reply:
Good point. Actually, if we have the `...` the enum shouldn't be all on one line. I'll update the PR.